src/dialog/marisa.c

```diff
@@ -115,6 +317,34 @@ DIALOG_TASK(marisa, Stage4PostBoss) {
 	VARIANT(kurumi, defeated);
 	FACE(kurumi, defeated);
 
+	FACE(marisa, smug);
+	MSG(marisa, "Whoever has the best petticoat wins. It’s a scientific fact.");
+	FACE(marisa, sweat_smile);
+	MSG(marisa, "… but since when do I care about scientific facts?");
+
+	MSG(kurumi, "W-wait, what was I doing again?");
+	MSG(kurumi, "(And didn’t I see this nerdy looking girl before…?)");
+
+	FACE(marisa, normal);
+	MSG(marisa, "Ah, sorry, sorry. I think ya slipped and hit your head.");
+	MSG(marisa, "I’m Gensōkyō’s… petticoat inspector.");
+	MSG(marisa, "I’m here to inspect the petticoats in this here mansion.");
```

Comment:
I'm not sure if this is a typo or intentional.

```suggestion
	MSG(marisa, "I’m here to inspect the petticoats in this mansion.");
```

Reply:
Intentional. It's meant to mimic a kind of casual southern drawl, since I apparently imagine Marisa slipping into kansai-ben on occasion when she's trying to sound slick.

Reply:
Alright, in that case it's fine.